docs/asset-store/docs/05-01-asset-store.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Asset Store chart
+type: Configuration
+---
+
+To configure the Asset Store chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.
+
+To learn more about how to use overrides in Kyma, read [this document](/root/kyma/#configuration-helm-overrides-for-kyma-installation). See also examples of [top-level charts overrides](/root/kyma/#configuration-helm-overrides-for-kyma-installation-top-level-charts-overrides) and [sub-charts overrides](/root/kyma/#configuration-helm-overrides-for-kyma-installation-sub-chart-overrides).
```

Comment:
if we keep those links in one config file only it will be hard for you to create a specific template
maybe let's make it a bullet point list inside a note and then it will be embedded in each document?

Reply:
OK. I created a tip with links in bullet points. In the config docs template, you have a general link to Helm overrides in Kyma and then you choose between linking to top-level charts overrides or sub-chart overrides.